app/controllers/referee_interface/reference_controller.rb

```diff
@@ -3,19 +3,42 @@ class ReferenceController < ActionController::Base
     include LogQueryParams
     before_action :add_identity_to_log
     before_action :check_referee_has_valid_token
+    before_action :set_token_param
```

Comment:
Is this necessary? We call `params[:token]` in other parts of the controller.

We could do:

```ruby
def token_param
  params[:token]
end
```

And use that?

Reply:
I think it is necessary since we use it in the views too:
For example:

```
  <%= govuk_button_link_to 'Finish', referee_interface_reference_finish_path(token: @token_param) %>

```